app/v6/deletePropertyAnnualSubmission/DeletePropertyAnnualSubmissionService.scala

```diff
@@ -47,10 +48,11 @@ class DeletePropertyAnnualSubmissionService @Inject() (connector: DeleteProperty
     )
 
     val extraTysErrors = Map(
-      "INVALID_INCOMESOURCE_ID" -> BusinessIdFormatError,
-      "INVALID_CORRELATION_ID"  -> InternalError,
-      "NOT_FOUND"               -> NotFoundError,
-      "TAX_YEAR_NOT_SUPPORTED"  -> RuleTaxYearNotSupportedError
+      "INVALID_INCOMESOURCE_ID"  -> BusinessIdFormatError,
+      "INVALID_CORRELATION_ID"   -> InternalError,
+      "NOT_FOUND"                -> NotFoundError,
+      "TAX_YEAR_NOT_SUPPORTED"   -> RuleTaxYearNotSupportedError,
+      "OUTSIDE_AMENDMENT_WINDOW" -> RuleOutsideAmendmentWindowError
```

Comment:
This error is a downstreamError and therefore goes in the downstreamErrors val instead